FreeRTOS/FreeRTOS-Plus-TCP
# Update Atmel SAME5x port

Description
-----------
This PR updates the Atmel SAME5x port with endpoint and IPv6 related changes in the stack. The changes are:

- Update port APIs to latest
- Add  `pxATSAM5x_FillInterfaceDescriptor()`
- Enable multicast and unicast MAC address hash table on GMAC
- Add multicast and unicast MAC addresses to hash table to allow traffic including all nodes IPv6 multicast MAC

Test Steps
-----------
Tested with demo for:

- DHCP
- UDP
- ICMP
- RA
- DNSv4
- DNSv6

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
